pkg/apis/policy/types.go

```diff
@@ -215,6 +208,24 @@ type PodSecurityPolicySpec struct {
 	// is allowed in the "Volumes" field.
 	// +optional
 	AllowedFlexVolumes []AllowedFlexVolume
+	// AllowedUnsafeSysctls is a list of explicitly allowed unsafe sysctls, defaults to none.
```

Comment:
follow-up: we should maybe also mention that the kubelet has to whitelist them explicitly.

Reply:
follow-up: mention that these must not be forbidden.

Reply:
It does not feel it belongs here. In future, nodes can have different mechanism how to whitelist the unsafe sysctls, e.g. via configmap. It would mean to regen the docs and other pieces. I would rather mention that in the documentation.

Reply:
can be formulated in a neutral way, like: "It depends on the node to accept these unsafe sysctl as well".

Reply:
I will mention it.

Reply:
Done